door43_tools/templaters.py

```diff
@@ -67,16 +81,16 @@ def apply_template(self):
         template = BeautifulSoup(self.template_html, 'html.parser')
 
         # find the target div in the template
-        content_div = template.body.find('div', {'id': 'content'})
+        content_div = template.body.find('div', {'id': 'outer-content'})
```

Comment:
This might fix one of the problems I was having with the floating header. Thanks!